docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -241,6 +241,7 @@ package:
 | `http://tizen.org/feature/network.wifi.direct.display` | Specify this key, if the application requires the Wi-Fi Direct display feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.direct.service_discovery` | Specify this key, if the application requires the Wi-Fi Direct service discovery feature. | 2.3   |
 | `http://tizen.org/feature/network.wifi.tdls` | Specify this key, if the application requires the Wi-Fi Tunneled Direct Link Setup (TDLS). | 3.0   |
+| `http://tizen.org/feature/network.wifi.mac_randomization` | Specify this key, if the application requires the MAC randomization feature. | 6.5   |
```

Comment:
@omar-srbd In this kind of table, how about following the alphabetical order?

Reply:
@mijin85cho, sure.
We'll fix the alphabetical order for the feature keys with a new PR in the near future. (:

Reply:
Done.